.github/workflows/ci.yml

```diff
@@ -42,32 +39,20 @@ jobs:
 
         # Use mamba for Windows dependencies
         - uses: mamba-org/setup-micromamba@v1
-          if: contains(matrix.os, 'windows')
+          if: matrix.os == 'windows-latest' || matrix.os == 'macOS-latest'
```

Comment:
The main advantage of `contains` is that it work fine for both `windows-latest` or `windows-2019`. Any reason to switch to exact match?

Reply:
I was just trying to be coherent with the rest of the file, and since we are currently using only `-latest` OS it made no difference. Anyway, we can switch all the instances to use `contains` rater then `==`

Reply:
No problem, I was just wondering!